core/client/pom.xml

```diff
@@ -79,6 +79,11 @@
       <artifactId>alluxio-underfs-s3</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-underfs-swift</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
We originally left this out because of the dependency requirement on the hadoop version. Has that issue been resolved?